EasyCommands/CommandParsers/ParameterParsers.cs

```diff
@@ -444,30 +444,28 @@ partial class Program {
                 u => u.Replace(" : ", " :: ")
                     .Split(new[] { ' ' }, StringSplitOptions.RemoveEmptyEntries)
                     .SelectMany(v => firstPass(v))
-                    .Select(v => new Token(v, false, false))
-                    .ToArray())
+                    .Select(v => new Token(v, false, false)))
             .ToList();
         }
 
-        Token[] TokenizeEnclosed(String token, string characters, Func<String, Token[]> parseSubTokens) =>
+        IEnumerable<Token> TokenizeEnclosed(String token, string characters, Func<String, IEnumerable<Token>> parseSubTokens) =>
             characters.Length == 0 ? parseSubTokens(token) :
                 token.Trim().Split(characters[0])
                 .SelectMany((element, index) => index % 2 == 0  // If even index
-                    ? TokenizeEnclosed(element, characters.Remove(0,1), parseSubTokens)  // Split the item
-                    : new Token[] { new Token(element, true, characters.Length > 1) })  // Keep the entire item
-                .ToArray();
+                    ? TokenizeEnclosed(element, characters.Remove(0, 1), parseSubTokens)  // Split the item
+                    : Once(new Token(element, true, characters.Length > 1)));  // Keep the entire item
```

Comment:
I see Once() several times. Would NewList() work here to save 3 chars per usage?

Reply:
see above